### ISSUE TITLE
Need team data for team events

### ISSUE THREAD

**Issue by duxbuse** [duxbuse/ninthage-data-analytics#21]
# tldr;
Basically we need to measure team performance separately individual game data is not sufficient

## more details
On Team Tourneys there usually are quite big differences. Why you ask? Well for example one army is a set army (a point denial list designed to take everything and to tie or loose as small as possible) the other one is a counter army (designed to counter builds like gunlines or monster mash) and other ones are scorers (designed to get as much points as possible to win the battle while counters and set armies paralyse the opponents).

And there also is pairing strategy. For example a certain list might be paired in a way that it plays 7-13 all 8 games. Paired differently it might have achieved 12-8 in all games but vs the other armies in his team would have played 5-15 vs. his opponent and played 14-6 vs. the opponents they actually played.

So even an army which individual performed poor may have been very important to achieve the team success as it took all the difficult matches for the other armies.

All those and other factors make the final placing much = the team success more meaningful than the individual successes.

So we award all armies which were in a team what the team as a whole earned.

(Number of teams - Ranking of the Team) / (Number of Teams -1) is what we give to each army in the same team.
